tests/config/test_section_serialization.py

```diff
@@ -93,6 +94,7 @@ def test_read_configuration_file():
 root_folder = "./taipy/"
 storage_folder = ".data/"
 clean_entities_enabled = "True:bool"
+repository_type = "filesystem"
```

Comment:
I would add some values in repository_properties to test the serialization/deserialization of such a dictionary attribute.